packages/data/src/plugins/resolvers.js

```diff
@@ -89,3 +92,51 @@ export function* getJetpackConnectUrl( query ) {
 
 	yield setIsRequesting( 'getJetpackConnectUrl', false );
 }
+
+export function* getPaypalOnboardingStatus() {
+	yield setIsRequesting( 'getPaypalOnboardingStatus', true );
+
+	const errorData = yield select(
+		STORE_NAME,
+		'getPluginsError',
+		'getPaypalOnboardingStatus'
+	);
+	if ( errorData && errorData.data && errorData.data.status === 404 ) {
+		// The get-status request doesn't exist fall back to using options.
+		yield setOnboardingStatusWithOptions();
+	} else {
+		try {
+			const url = PAYPAL_NAMESPACE + '/onboarding/get-status';
+			const results = yield apiFetch( {
```

Comment:
This is failing on the latest `obw-work` branch of the plugin. I'm using PHP 7.2 which I think could be having an effect. It's giving a 500 error, saying it can't find `Onboarding_REST_Controller`.

Reply:
Did this happen when switching from the `trunk` to `obw-work` branch? I believe that just requires a `composer dumpautoload`.
> You need to re-generate the autoloader files so that it picks up the new class.

Reply:
Please see https://github.com/woocommerce/woocommerce-admin/pull/6261#issuecomment-774003966